### DISCUSSION TITLE
저희 Organization에 member들이 추가 되었으면 좋겠습니다

### DISCUSSION THREAD

**Discussion by chadseok** [wanted-pre-onboarding-side-project/habit-tracker#27] [Ideas]
어떤 멤버로 구성되어있는지 나타내는 것이 좋아보입니다

organziation에 member 추가하는 방법:
https://docs.github.com/ko/organizations/managing-membership-in-your-organization/inviting-users-to-join-your-organization

**Comment by greyHairChooseLife** [comment]
어떻게 나타내면 좋을까요? 저는 README에 적으면 좋겠다고 생각은 했었어요.

**Reply by chadseok** [reply]
아뇨 링크로 들어가보시면 저희 org에 멤버 추가하는 방법이 나와있습니다

**Reply by jiyeon2** [reply]
창환님이 첨부해주신 링크대로 창환님을 초대했습니다! 확인 부탁드릴게요~

저랑 상연님은 이미 팀 멤버로 추가되어 있었는데 표시가 안되는 걸까요??
그리고 초반에 두 사람이 이것저것 설정을 시도하느라
저랑 상연님이 둘 다 owner인 상태입니다

한 사람만 owner 역할을 가지고 있는 게 안전하다고 생각하는데
상연님만 owner로 설정해두고
논의를 통해 결정된 repository 변경사항은 상연님이 반영해주시는 건 어떨까요?

**Reply by greyHairChooseLife** [reply]
좋습니다. 

저는 4월 대부분의 시간 디스코드에 접속 해 있을 예정이라 대응하기 좋을 것 같습니다.

혹시라도 관련 불편 사항이 발생하면 논의 후 조정하도록 해요.

---

방금 [role](https://github.com/orgs/wanted-pre-onboarding-side-project/people) 조정했습니다.

@chadseok :point_right: `member, maintain`
@jiyeon2  :point_right: `member, maintain`
@greyHairChooseLife :point_right: `owner, admin`

>Maintain role 설명
_Can read, clone, and push to this repository. They can also manage issues, pull requests, and some repository settings._

이 변화는 실수로 리포지토리 또는 protected 브랜치의 삭제하는 것을 방지하기 위한 조치입니다. 모든 멤버는 모든 브랜치에 PR을 리뷰하고 승인 할 권리가 있어야 정상입니다.

더불어 기존 리포지토리 PR 승인을 승인 리뷰 개수를 2개로 조정했습니다.
